specs/clipboard-api/clipboard.idl

```diff
@@ -0,0 +1,426 @@
+  /// </summary>
+  [contract(ClipboardContract, 1.0)]
+  runtimeclass DataPackagePropertySetView :
+    [default] IDataPackagePropertySetView
+    , Windows.Foundation.Collections.IMapView<String, Object>
+    , Windows.Foundation.Collections.IIterable<Windows.Foundation.Collections.IKeyValuePair<String, Object> >
+  { }
+
+  [contract(ClipboardContract, 1.0)]
+  [exclusiveto(DataPackagePropertySetView)]
+  [hasvariant]
+  interface IDataPackagePropertySetView requires
+    Windows.Foundation.Collections.IMapView<String, Object>
+    , Windows.Foundation.Collections.IIterable<Windows.Foundation.Collections.IKeyValuePair<String, Object> >
+  {
+    String Title { get; };
+    String Description { get; };
+    Windows.Storage.Streams.IRandomAccessStreamReference Thumbnail { get; };
+    Windows.Foundation.Collections.IVectorView<String> FileTypes { get; };
+    String ApplicationName { get; };
+    Windows.Foundation.Uri ApplicationListingUri { get; };
+
+    String PackageFamilyName { get; };
+    Windows.Foundation.Uri ContentSourceWebLink { get; };
+    Windows.Foundation.Uri ContentSourceApplicationLink { get; };
+    Windows.Storage.Streams.IRandomAccessStreamReference Square30x30Logo { get; };
+    Windows.UI.Color LogoBackgroundColor { get; };
+
+    String EnterpriseId { get; };
+
+    String ContentSourceUserActivityJson { get; };
+
+    Boolean IsFromRoamingClipboard { get; };
+  }
+
+  /// <summary>
+  /// An acknowledgement by the system that your app is in the process of
+  /// a delay rendering, with the ability to signal the system when done.
+  /// </summary>
+  [contract(ClipboardContract, 1.0)]
+  runtimeclass DataProviderDeferral
+  {
+    void Complete();
+  }
+
+  /// <summary>
+  /// Input to a <see cref="DataProviderHandler"/> function for delay-rendering.
+  /// </summary>
+  [contract(ClipboardContract, 1.0)]
+  runtimeclass DataProviderRequest
+  {
+    String FormatId { get; };
+    Windows.Foundation.DateTime Deadline { get; };
+    DataProviderDeferral GetDeferral();
+    void SetData(Object value);
+  }
+
+  /// <summary>
+  /// Translates strings containing fragments of HTML to and from the special format
+  /// used by <see cref="StandardDataFormats.Html"/>, described here:
+  /// https://docs.microsoft.com/windows/win32/dataxchg/html-clipboard-format
+  /// </summary>
+  static runtimeclass HtmlFormatHelper
+  {
+    static String GetStaticFragment(String htmlFormat);
+    static String CreateHtmlFormat(String htmlFragment);
+  }
+
+  /// <summary>
+  /// Input parameters for <see cref="DataPackage.OperationCompleted"/> event handler functions.
+  /// </summary>
+  [contract(ClipboardContract, 1.0)]
+  runtimeclass OperationCompletedEventArgs
+  // FIXME: since the namespace is now ApplicationModel, the name OperationCompletedEventArgs
+  // is too generic. Should we rename it?
+  {
+    DataPackageOperation Operation { get; };
+
+    String AcceptedFormatId { get; };

```

Comment:
This property is really poorly named, and [the documentation](https://docs.microsoft.com/en-us/uwp/api/windows.applicationmodel.datatransfer.operationcompletedeventargs.acceptedformatid?view=winrt-19041) does little more than repeat the name. I should investigate what the property even does before retaining it.

My best guess before doing that investigation is that on a paste (Clipboard.GetContent() followed by DataPackageView.GetDataAsync(format) on the return value from Clipboard), the DataPackage.OperationCompleted event is signaled, with this property set to the name of the data format that the target app read out.